frontend/src/App.jsx

```diff
@@ -1,12 +1,33 @@
+import { useState } from "react";
 import Home from "./pages/Home";
 
 import "./App.css";
 
 function App() {
+  const [fetchEvents, setFetchEvents] = useState(null);
```

Comment:
fetchedEvents ? or just events ? (and calling the `events` in fetch `fetchedEvents`)